Django-CoWork-Connect/coapp/models.py

```diff
@@ -1,3 +1,9 @@
 from django.db import models
 
-# Create your models here.
+class Oferta(models.Model):
+    title = models.CharField(max_length=100)
+    descriptions = models.TextField()
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+
+    def __str__(self):
+        return self.tytul
```

Comment:
masz tutaj błęd wiec wychodzi na to ze w ogole nie sprawdziels kodu i nawet nie stworzyles 1 obiektu klasy Oferta